tests/config/junos/BGP_NEIGHBOR_STATE_CHANGED/active_to_connect/yang.json

```diff
@@ -0,0 +1,39 @@
+{
+  "yang_message": {
+    "bgp": {
+      "neighbors": {
+        "neighbor": {
+          "1.1.1.1": {
+            "state": {
+              "peer_as": "2222",
+              "session-state-change-event": "AnEvent",
```

Comment:
Is this a genuine event message (`AnEvent`)? If not, could you please reference a real example (same documentation reason).

Reply:
Again, just me being lazy, will fix. Also this relates to my comments on the ospf events.